roles/openshift_repos/tasks/main.yaml

```diff
@@ -40,4 +40,21 @@
     - openshift_deployment_type == 'origin'
     - openshift_enable_origin_repo | default(true) | bool
 
+  # Singleton block
+  - when: r_osr_first_run | default(true)
+    block:
+    - name: Ensure clean repo cache in the event repos have been changed manually
```

Comment:
I left it as a handler because the previous tasks will trigger it only if they change something.  I'm guessing there is a reason why we call openshift_repos 39 times, so I was not going to just make it not do anything on subsequent runs.  Maybe we need to step back and understand why it is being called so many times in the first place.  The original intent here was to clear the cache at least the first time in case there were manual changes just prior to the playbook being run.